src/main/resources/avro/references.avdl

```diff
@@ -81,21 +81,12 @@ record ReferenceSet {
   string id;
 
   /**
-  The IDs of the `Reference` objects that are part of this set.
-
-  This field should be left null if the set contains a very large number of
-  `Reference`s. The `Reference`s in the set may also be obtained through a
```

Comment:
Jerome Kelleher <notifications@github.com> writes:
> Yeah, you're right. I suppose what we're lacking is the 'big picture'
> documentation, but I think that's underway at UCSC.

Underway, but delayed by
  (1) need to updated to match the new master
  (2) a European who seems to think August is for holidays

it will still be a work in progress